gravitee/values.yaml

```diff
@@ -209,14 +211,18 @@ api:
       memory: 512Mi
 
 gateway:
+  enabled: true
   type: Deployment
   name: gateway
   logging:
     debug: false
   replicaCount: 2
+  sharding_tags: default
+  tenant:
```

Comment:
no value? no error?

Reply:
I would advise to remove these to lines because if the gateway is started on the default sharing tags, you have to register this one as part of the management api. Without this, you will not be able to deploy apis to gateway